src/matlab2tikz.m

```diff
@@ -1519,33 +1539,29 @@ case guitypes()
     % a reasonable default.
     matlabDefaultNumMinorTicks = 3;
     if ~isempty(ticks)
-        options = opts_add(options, ...
-            [axis,'tick'], sprintf('{%s}', ticks));
+        options = opts_add(options, [axis,'tick'], sprintf('{%s}', ticks));
```

Comment:
The newline is really legacy stuff, probably when monitors were still not wide enough. I asked my self several times why we split it in the first place, but you know, there really is no incentive in beautyfying the whole code, unless you are already working on it.